recipes/wildmeshing/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "wildmeshing" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wildmeshing/wildmeshing-python/archive/{{ version }}.tar.gz
+  sha256: 83b9fe8f1d0e6565238a9bfded2a112502f98611a671c938d6c5e7a59a45574f
+
+build:
+  number: 0
+  skip: True  # [py<31]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+    - libblas
+    - libcblas
+    - numpy
+    - gmp     # [unix]
+    - mpir    # [win]
+  run:
+    - python
+    - mpir    # [win]
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - wildmeshing
+
+about:
+  home: https://github.com/wildmeshing/wildmeshing-python
+  license: MPL-2.0
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
@teseoch, this needs to point to a file with both the license for wildmeshing and for the bundled dependencies.

Reply:
that repo contains a LICENCE and the other one bundled, is it not enough?

that entry is not for the "website"?

Reply:
> that entry is not for the "website"?

No, conda-build will check for this file in the source tarball and package it in the conda package as `info/LICENSE.txt`.

Both LICENSE and LICENSE_bundled needs to be packaged in the conda package. When you have it like this, only LICENSE is packaged in the conda package.

Reply:
@teesoch, this is not resolved yet

Reply:
@teseoch, can you fix this?

Reply:
I dont know how. How can i force both files to get packaged?

Reply:
scipy on forge dosent do it (@isuruf )

Reply:
You'll have to modify the "script" section above to copy over both licenses into the install location or merge the two licenses into a single license.

Reply:
ok I merge both. Is there anything else?

Reply:
It looks like you're not including the license text from the other pages and so far as I know a URL doesn't count for most common licences.

Using SciPy as an example, they concatenate the bundle file with their licence when packaging the sources and the full text is included in the file or the referenced file is included. To see this locally (change to your Conda installation as appropriate):
```bash
less ~/miniconda3/pkgs/scipy-*/info/LICENSE.txt
```
for `L-BFGS-B` it references this file which is also included in the package:
```bash
less ~/miniconda3/pkgs/scipy-*/lib/python*/site-packages/scipy/optimize/lbfgsb_src/README
```

Reply:
I did concatenate all licences together in one unique one.

I was also told that there is no need to host the different licences and the link was enough since they are BSD / MIT

Reply:
Oh yeah, you did, in the upstream package.  Ok!